aws/sdk/examples/snowball/src/bin/create-address.rs

```diff
@@ -5,14 +5,14 @@
 
 use aws_sdk_snowball::model::Address;
 use aws_sdk_snowball::{Config, Region};
-use aws_types::region::{EnvironmentProvider, ProvideRegion};
+use aws_types::region;
 use structopt::StructOpt;
 
 #[derive(Debug, StructOpt)]
 struct Opt {
-    /// The region
+    /// The default region
     #[structopt(short, long)]
-    region: Option<String>,
+    default_region: Option<String>,
```

Comment:
question

Why change this to be prefixed with `default_`? Seems like if set it is the region used.

Reply:
That's a good point. Going to change it over.